docs/integrations/aa/pimlico.md

```diff
@@ -228,8 +228,22 @@ We will now generate a wallet that can act a regular Ethers.js wallet, but will
 const pkpWallet = new PKPEthersWallet({
   pkpPubKey: pkp[pkp.length - 1].publicKey,
   rpc: "<standard RPC URL for the chain you are using>", // e.g. https://rpc.ankr.com/eth_goerli
+  litNodeClient,
+  authContext: {
+    getSessionSigsProps: {
+      chain: 'ethereum',
+      expiration: new Date(Date.now() + 60_000 * 60).toISOString(),
+      resourceAbilityRequests: resourceAbilities,
+      authNeededCallback,
+    },
+  },
+  // controllerSessionSigs: sesionSigs, // (deprecated) If you will be passing sessionSigs directly, do not pass authContext
```

Comment:
I don't believe `controllerSessionSigs` is deprecated. Tried both and `authContext` allows for the `getSessionSigsProps` while `controllerSessionSigs` still allows for a `sessionSigs` object. But let me know if this has been discussed already